test/libyul/yulOptimizerTests/expressionSimplifier/shift_left_overflow.yul

```diff
@@ -0,0 +1,14 @@
+{
+  function foo(x)
+  {
+    mstore(0, shl(1,shl(not(0),x)))
+  }
+}
+// ----
+// expressionSimplifier
+// {
+//     function foo(x)
+//     {
+//         mstore(0, shl(256, x))
+//     }
+// }
```

Comment:
This test may fail because it is picked up by an IPC test involving aleth-homestead.

Reply:
Ah right, I forgot to remove them. Thanks!